spec/models/skin_parent_spec.rb

```diff
@@ -62,6 +62,14 @@
       expect(duplicate_ancestor.errors[:base]).not_to be_empty
     end
 
-  end
+    it "must not allow a parent to be blank" do
```

Comment:
Would you mind changing the name of this test to reflect the fact that it's the title of the skin that's blank, not the skin itself?